1.Exploratory_Data_Analysis/notebooks/12.visualize_correlations.ipynb

```diff
@@ -45,57 +45,64 @@
                             "\\hline\n",
-                            "\t ER   & Mito & 0.3065211 & BR00117004\\_normalized\\_sc\\_agg & ER\\_Mito \\\\\n",
-                            "\t ER   & DNA  & 0.2486484 & BR00117004\\_normalized\\_sc\\_agg & ER\\_DNA  \\\\\n",
-                            "\t ER   & AGP  & 0.8950753 & BR00117004\\_normalized\\_sc\\_agg & ER\\_AGP  \\\\\n",
-                            "\t ER   & RNA  & 0.8819529 & BR00117004\\_normalized\\_sc\\_agg & ER\\_RNA  \\\\\n",
-                            "\t Mito & DNA  & 0.2764739 & BR00117004\\_normalized\\_sc\\_agg & Mito\\_DNA\\\\\n",
-                            "\t Mito & AGP  & 0.1935663 & BR00117004\\_normalized\\_sc\\_agg & Mito\\_AGP\\\\\n",
+                            "\t ER   & Mito & 0.7198829 & BR00117004\\_normalized\\_sc\\_agg & A01 & ER\\_Mito \\\\\n",
+                            "\t ER   & DNA  & 0.5179363 & BR00117004\\_normalized\\_sc\\_agg & A01 & ER\\_DNA  \\\\\n",
+                            "\t ER   & AGP  & 0.9162101 & BR00117004\\_normalized\\_sc\\_agg & A01 & ER\\_AGP  \\\\\n",
+                            "\t ER   & RNA  & 0.9632002 & BR00117004\\_normalized\\_sc\\_agg & A01 & ER\\_RNA  \\\\\n",
+                            "\t Mito & ER   & 0.7198829 & BR00117004\\_normalized\\_sc\\_agg & A01 & Mito\\_ER \\\\\n",
+                            "\t Mito & DNA  & 0.1735317 & BR00117004\\_normalized\\_sc\\_agg & A01 & Mito\\_DNA\\\\\n",
                             "\\end{tabular}\n"
                         ],
                         "text/markdown": [
                             "\n",
-                            "A tibble: 6 \u00d7 5\n",
+                            "A tibble: 6 \u00d7 6\n",
                             "\n",
-                            "| channel1 &lt;fct&gt; | channel2 &lt;fct&gt; | correlation &lt;dbl&gt; | plate &lt;chr&gt; | correlation_label &lt;chr&gt; |\n",
-                            "|---|---|---|---|---|\n",
-                            "| ER   | Mito | 0.3065211 | BR00117004_normalized_sc_agg | ER_Mito  |\n",
-                            "| ER   | DNA  | 0.2486484 | BR00117004_normalized_sc_agg | ER_DNA   |\n",
-                            "| ER   | AGP  | 0.8950753 | BR00117004_normalized_sc_agg | ER_AGP   |\n",
-                            "| ER   | RNA  | 0.8819529 | BR00117004_normalized_sc_agg | ER_RNA   |\n",
-                            "| Mito | DNA  | 0.2764739 | BR00117004_normalized_sc_agg | Mito_DNA |\n",
-                            "| Mito | AGP  | 0.1935663 | BR00117004_normalized_sc_agg | Mito_AGP |\n",
+                            "| channel1 &lt;fct&gt; | channel2 &lt;fct&gt; | correlation &lt;dbl&gt; | plate &lt;chr&gt; | well &lt;chr&gt; | correlation_label &lt;chr&gt; |\n",
+                            "|---|---|---|---|---|---|\n",
+                            "| ER   | Mito | 0.7198829 | BR00117004_normalized_sc_agg | A01 | ER_Mito  |\n",
```

Comment:
How are you calculating these correlations? Is this a single correlation feature? How did you chose the feature? If it's a summary of multiple feature correlations, how did you summarize?